verilog/analysis/verilog_linter_configuration_test.cc

```diff
@@ -788,6 +788,28 @@ TEST(RuleBundleTest, ParseRuleBundleSkipComments) {
   }
 }
 
+TEST(RuleBundleTest, ParseRuleBundleIgnoreExtraComma) {
+  // Multiline rules might still have a comma from the one-line
+  // rule configuration. They shouldn't harm.
+  const std::string text =
```

Comment:
Looks like ParseConfiguration accepts a string_view, so this could be constexpr absl::string_view probably.

Reply:
done..